### PR TITLE
multiple changes and removed extra spaces

### PR DESCRIPTION
**Pull request by priya-sudarshanam**
@rbran100
a)reformatted the output for most-viewed-pages and sites including the number of results - Dan's request
b)made the start and end-date past 1-day/24 hours earlier as per Ferdi's request
c)added comments and added variables
d)removed spaces and reindented

### CODE REVIEW

**Comment by priya-sudarshanam** [comment]
@rbran100 
Added real-time code to the .module and js.tpl file. can you pls merge this when you get a chance..thx

File: ossap_stats/ossap_stats.info [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "OSSAP Stats"
-description = "Statistics for OpenScholar Single Access Point (OSSAP)"
+description = "Statistics really for OpenScholar Single Access Point (OSSAP)"
```
Comment (rbran100) [inline ossap_stats/ossap_stats.info]: You don't need this really here, or the removal of the dependency.  Can you re-open without.

**Comment by rbran100** [comment]
Just that one correction and this will be ready to merge, also when you make a new PR can you make sure it is up to date with the main ossap branch (git merge the main branch).

**Comment by rbran100** [comment]
re-opening till there is a new PR

**Comment by priya-sudarshanam** [comment]
@rbran100 
a)merged conflicts in the forked repo and made the forked repo up-to-date with the main branch
b)removed a function which was not being used
c)renamed the admin url to reflect the naming convention of ossap_stats

**Pull request merged**